include/swift/SIL/SILValue.h

```diff
@@ -586,6 +586,8 @@ class Operand {
   Operand(const Operand &use) = delete;
   Operand &operator=(const Operand &use) = delete;
 
+  Operand(Operand &&) = default;
```

Comment:
While you’re here, can you declare the move-assign operator?

Reply:
Sure, I'll do that as a separate change.